config/ember-try.js

```diff
@@ -1,6 +1,55 @@
 /*jshint node:true*/
 module.exports = {
-  // see package.json
-  // TLDR: 1.13.x ++
-  useVersionCompatibility: true
```

Comment:
We should retain the 1.13.x + [version compatibility](https://github.com/yapplabs/ember-buffered-proxy/pull/28/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L57) as it gives us more test coverage than this change.